pkg/cli/printhelper/printhelper.go

```diff
@@ -29,8 +29,8 @@ EXTERNAL-IP may be 'pending' for a while until your cloud provider has created a
 
 #########################################################################################################
 
-The command will wait until loft is reachable under the host. You can also abort and use port-forwarding instead
-by running 'loft start' again.
+The command will wait until vcluster platform is reachable under the host. You can also abort and use port-forwarding instead
```

Comment:
I'm basing this suggestion on [vcluster-docs](https://github.com/loft-sh/vcluster-docs/blob/main/.github/styles/Loft/no-vcluster-platform.yml) style guide, but you may want to confirm what we want to use in this context. I also think `vCluster Platform` might be acceptable.

```suggestion
The command will wait until the platform is reachable under the host. You can also abort and use port-forwarding instead
```

Reply:
We want to refer the the platform as `the platform` but since this is a CLI helper `vCluster platform` is also ok.